contributors/design-proposals/seccomp.md

```diff
@@ -112,14 +140,21 @@ specification of seccomp filters, with arguments, operators, and actions.
 Docker allows the specification of a single seccomp filter.  There are
 community requests for:
 
-Issues:
-
 * [docker/22109](https://github.com/docker/docker/issues/22109): composable
   seccomp filters
 * [docker/21105](https://github.com/docker/docker/issues/22105): custom
   seccomp filters for builds
 
-#### rkt / appcontainers
+Implementation details:
```

Comment:
This is good stuff; you may want to split the updates to the content out into a separate PR, since they're non-controversial and can easily be merged without depending on the new content here being accepted.